src/framework/auth/services/auth.service.ts

```diff
@@ -3,12 +3,13 @@
  * Copyright Akveo. All Rights Reserved.
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
-import { Injectable, Optional, Inject, Injector } from '@angular/core';
+import { Inject, Injectable, Injector, Optional } from '@angular/core';
 
 import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/switchMap';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/do';
+import { switchMap } from 'rxjs/operators/switchMap';
+import { map } from 'rxjs/operators/map';
+import { tap } from 'rxjs/operators/tap';
+import { of } from 'rxjs/observable/of';
```

Comment:
Could you do it like this `import {switchMap, map, tap, of} from 'rxjs/operators'`?